### ISSUE TITLE
no way to hide hints for new users

### ISSUE THREAD

**Issue by foss-** [friendica/friendica#4776]
### Expected behavior

There should be an option to hide the links for news users after signing up on friendica. Not sure if this is limited to the instance tested on (squeet.me) or a general friendica issue or maybe even limited to the frio theme. Adding a small "x" in the top right of the new user info details would solve this. Clicking that "x" should close the info and not display the new user info again. So also the three dots in the top left menu should then disappear.

### Actual behavior

No "x" or other option to hide the new user info.

![new user info](https://user-images.githubusercontent.com/5708172/38458026-e24f0b6a-3a98-11e8-96a5-635c1965d91d.jpg)

**Comment by karbovskibg** [comment]
include there is help menu from right that everybody can spy what peoples have installed on they friendico. include after update messaes can't be edited, include after database update compose new message dissapear. with old versions from 3.5 it didn;t happend.

**Comment by MrPetovan** [comment]
@karbovskibg Your comment doesn't make much sense, "what people have installed on their Friendica node" usually isn't a secret. Which messages can't be edited? Your posts? Please open a separate issue for this.

**Comment by AlfredSK** [comment]
The message for new users disappears after 14 days. The possibility to remove the new member widget may be implemented.
But the left aside menu is important. It makes no sense to remove/hide it.

**Comment by foss-** [comment]
@AlfredSK Important in what way? I understand the current logic you describe (show menu for 14 days). But what if a user reads all suggested new user documentation on day 1, has understood it and has no need to see it anymore? I think it's a good idea to allow users to hide it. Or we assume user stupidity and that they may hide stuff they later need. But then the new user documentation should still be accessible via the help section (is that currently the case?). As long as the latter is the case I'd still suggest to add an option to hide new user docu even before 14 days have passed.

**Comment by karbovskibg** [comment]
petovan then open your house so everybody can enter and see what you have in your house.

**Comment by foss-** [comment]
Ok, clearly then that menu should be kept. Have not used friendica a lot on mobile and on the desktop version I have not seen the three dots in use.

**Comment by annando** [comment]
I guess we can implement some setting in the user settings "don't show new user hint". If a user finds this, the person shouldn't need the hint anymore ;-)

**Comment by foss-** [comment]
Sounds good, but why not just had an "x" to that view from the screenshot? Isn't that the cross platform icon to close / hide a message that all users would be familiar with? That would be displayed at the right location where a user would expect the hide option without having to dive into unknown settings sub-menus. As an alternative there could be a "Hide" text / button.

**Comment by rabuzarus** [comment]
Yes a "x" would be fine. We can solve this with js. But nevertheless we need a user setting to to re-enable it again. And the js part would  need a module (user settings) where it can `post` the data to disable the widget.

**Comment by ntelogo** [comment]
hey, 
it's been a while for this issue. Has there been any changes? What's the way to hid the 'new member'  side menu?
thnx!
